src/sass/layout/_film-card.scss

```diff
@@ -3,12 +3,12 @@
   grid-template-columns: repeat(1, 280px);
   justify-content: center;
   grid-gap: 20px;
-  margin-top: 20px;
+  // margin-top: 20px;
```

Comment:
Why is the margin removed?

Reply:
because the padding from the header is set by the container, and this is an additional marigin that is not in the layout